src/main/scala/dpla/ingestion3/enrichments/Spatial.scala

```diff
@@ -0,0 +1,172 @@
+ * geo:lat value that contains both lat and long -- not sure if that's something
+ * that's been fixed. It's technically a data-cleanup issue.
+ *
+ * [ ... abridged ... ]
+ * @prefix dc: <http://purl.org/dc/terms/> .
+ * @prefix dpla: <http://dp.la/about/map/> .
+ * @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+ * @prefix geo: <http://www.w3.org/2003/01/geo/wgs84_pos#> .
+ * @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+ * [...]
+ * <http://ldp.staging.dp.la/ldp/items/00ea7274bf61874a7b8df90eb0042ef0> a <http://www.w3.org/ns/ldp#Resource>,
+ * [...]
+ *   dc:spatial [
+ *    a edm:Place;
+ *    dpla:providedLabel """
+ *     North and Central America
+ *     United States
+ *     Texas
+ *     Dallas
+ *     Dallas
+ *   """;
+ *    geo:lat "32.7833,-96.80";
+ *    skos:exactMatch "http://vocab.getty.edu/tgn/7013503"
+ *  ];
+ *
+ * ... Otherwise, the one other provider whose unenriched RDF I've examined,
+ * NARA, has one-line strings for the spatial statements:
+ *
+ *   dc:spatial [
+ *    a edm:Place;
+ *    dpla:providedLabel "Arlington (Va.)"
+ *  ];
+ *
+ * Example output enriched data from Ingestion 2, Cal. Digital Library:
+ *
+ *  dc:spatial [
+ *    a edm:Place;
+ *    dpla:providedLabel "Los Angeles (Calif.)";
+ *    gn:countryCode "US";
+ *    geo:lat 3.405223e1;
+ *    geo:long -1.1824368e2;
+ *    skos:closeMatch <http://id.loc.gov/authorities/names/n79021240>;
+ *    skos:exactMatch <http://sws.geonames.org/5368361/>;
+ *    skos:prefLabel "Los Angeles, CA, United States"
+ *  ];
+ *
+ * Note that Ingestion 2 (specifically, the Audumbla::CoarseGeocode enrichment)
+ * was bailing on the assignment of parent features, probably due to the fact
+ * that MAPv4 specifies them as URIs, which is inconvenient for us when it comes
+ * time to serialize the records into JSON-LD and have it indexed effectively
+ * by Elasticsearch, where you want strings for the parent features (E.g.
+ * "Los Angeles County" or "California"), not URIs. Our users search for those
+ * strings!
+ *
+ * We have work to do to determine if MAPv4 is actually the right thing, and
+ * whether we need a revision; or if we want to jump through hoops to resolve
+ * parent features as part of the process to writing JSON for Elasticsearch to
+ * index.
+ */
+
+/* Case classes for JSON parser ... */
+case class Coords(lat: Double, lng: Double)
+
+case class Bounds(ne: Coords, sw: Coords)
+
+case class Geometry(center: Coords, bounds: Bounds)
+
+case class Feature(geometry: Geometry, woeType: Int, displayName: String)
+
+case class Interpretation(feature: Feature)
+
+case class Result(interpretations: List[Interpretation])
+
+object Spatial extends MappingUtils with DefaultVocabularies {
+
+  implicit val formats = DefaultFormats // Formats for json4s
+  val log = Logger.getLogger(getClass.getName)
+
+  def enrich(m: Model): Model = {
+
+    val spatialModel = m.filter(null, rdf.`type`, edm.Place)
```

Comment:
Codacy is really checking the RDF4J API here, I'd say ignore. 

Although in the long run, maybe having utility functions that have more descriptive names and filter out the nulls might be a good idea for legibility.